rest-api/test/unit_test/offline_test/public_metrics_export_test.py

```diff
@@ -150,6 +150,14 @@ def _create_data(self):
               test='1SAL',
               confirmed=TIME))
 
+      sample_dao.insert(
```

Comment:
So in order for this to be useful, we'd need to have a participant that had everything for being a full participant except 1SAL, and confirm that this stored sample (with no confirmed date) doesn't make them a full participant.

@calbach I can't see in the code where we're doing detailed tests of the public metrics stuff... do we have some somewhere?

Reply:
havent done this

Reply:
You can remove this for now. No sense in adding the sample without a matching test.

Reply:
done, merging.